src/components/pages/Login.tsx

```diff
@@ -76,7 +76,7 @@ export default function Login() {
           </Typography>
           <Typography role='error-message'>
             {/* TODO Style Typography */}
-            {errorLoginUser ? `${errorLoginUser}` : ''}
+            {errorLoginUser && `${errorLoginUser}`}
```

Comment:
the previous style is more preferred 
```suggestion
            {errorLoginUser ? errorLoginUser : ''}
```

Reply:
https://reactjs.org/docs/conditional-rendering.html#inline-if-with-logical--operator
`Note that returning a falsy expression will still cause the element after && to be skipped but will return the falsy expression. In the example below, <div>0</div> will be returned by the render method.`

logical && sometimes do unexpected behavior.

Reply:
thanks, i made this change in all files